feat: add support for GraphQL endpoint (#189)
crowdin/crowdin-api-client-python: README.md

    EXTENDED_REQUEST_PARAMS = {"proxies": proxies}
```

> ### GraphQL API
> This library also provides possibility to use [GraphQL API](https://developer.crowdin.com/graphql-api/) (only for Crowdin Enterprise).

Reviewer: ```suggestion
### GraphQL API

This library also provides the possibility to use [GraphQL API](https://developer.crowdin.com/graphql-api/):
```
